src/lib/containers/FullWidthAlert.tsx

```diff
@@ -94,12 +94,24 @@ function FullWidthAlert(props: FullWidthAlertProps) {
           <Typography variant="headline3">{title}</Typography>
           <Typography variant="body1">{description}</Typography>
         </span>
-        {secondaryButtonText && secondaryButtonHref && (
+        {secondaryButtonText && onSecondaryButtonClick && (
           <a
             className="secondaryButton"
             target="_blank"
             rel="noopener noreferrer"
-            href={secondaryButtonHref}
+            onClick={
+              typeof onSecondaryButtonClick === 'function'
```

Comment:
Looking at this prop name, I would assume that it's always a function.  Maybe a comment on the prop to inform the caller that they can pass either a function or a URL string that will be used for the href?